multiuser/api/che-multiuser-api-resource/src/main/java/org/eclipse/che/multiuser/resource/api/usage/ResourceService.java

```diff
@@ -106,10 +109,28 @@ public ResourceUsageService(ResourceUsageManager resourceUsageManager) {
   })
   public List<ResourceDto> getUsedResources(@PathParam("accountId") String accountId)
       throws NotFoundException, ServerException {
-    return resourceUsageManager
+    return resourceManager
         .getUsedResources(accountId)
         .stream()
         .map(DtoConverter::asDto)
         .collect(Collectors.toList());
   }
+
+  @GET
+  @Path("details/{accountId}")
```

Comment:
Other methods in this Service have `accountId` as first URL fragment while this method defines `accountId` as the second one. I think it would be better to have similar URLs structure in all methods.

Reply:
agree